.github/workflows/build.yaml

```diff
@@ -1,7 +1,25 @@
 name: Build
 
 on:
+  workflow_call:
+    inputs:
+      refLsp4ij:
+        description: 'Lsp4ij ref'
```

Comment:
It will be good to add a more clear description

Reply:
Added a more clear description